### PR TITLE
Add private sourcemaps doc

### PR DESCRIPTION
**Pull request by mmcdaris**
Documents our private sourcemaps feature.

Provides instructions on how to generate and upload sourcemaps to Airbrake.

### CODE REVIEW

File: jekyll/_docs/installing-airbrake/private-sourcemaps.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+## Uploading sourcemaps
+
+Contining on with our above example, the next steps assume the minified version
+of `app.js` is publicly available at `https://example.com/app.min.js`.
+
+The bottom of the `app.min.js` file should have a comment with the name of the
+of the sourcemap.
+
+```
+js //# sourceMappingURL=app.min.js.map
```
Comment (vmihailenco) [inline jekyll/_docs/installing-airbrake/private-sourcemaps.md]: `js` is on wrong line

File: jekyll/_docs/installing-airbrake/private-sourcemaps.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+This feature is available for all paid accounts.
+
+## Generating sourcemaps
+
+Generating sourcemaps can be done with
+[uglifyjs](https://github.com/mishoo/UglifyJS2#cli-source-map-options) and
+[webpack](https://webpack.js.org/configuration/devtool/#production).  Here is an
+example to minify and create a sourcemap for an `app.js` file:
+
+```js
```
Comment (vmihailenco) [inline jekyll/_docs/installing-airbrake/private-sourcemaps.md]: this is shell I guess

**Comment by vmihailenco** [comment]
Overall looks good :+1:

**Comment by mmcdaris** [comment]
Thanks @vmihailenco, PTAL. I've pushed changes addressing the points you brought up, put a note at the top about this being a premium feature, and moved this doc to the features section.

File: jekyll/_docs/features/private-sourcemaps.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+categories: [features]
+description: Private sourcemaps
+---
+
+![](http://f.cl.ly/items/443E2J1D2W3x1E1u3j1u/JS-airbrakeman.jpg)
+
+**Private sourcemaps are a premium feature available on the Business Plus and
+Enterprise plans.**
+
+This guide assumes you have already [installed Airbrake in you JavaScript
```
Comment (thompiler) [inline jekyll/_docs/features/private-sourcemaps.md]: Typo: `you` -> `your`
>installed Airbrake in **your** JavaScript

File: jekyll/_docs/features/private-sourcemaps.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+  --source-map "root='https://example.com/',url='app.min.js.map'"
+```
+
+This command will create two new files, a minified `app.min.js` file and a
+sourcemap named `app.min.js.map`. The sourcemap file is what we will be
+uploading to Airbrake.
+
+## Uploading sourcemaps
+
+Contining on with our above example, the next steps assume the minified version
```
Comment (thompiler) [inline jekyll/_docs/features/private-sourcemaps.md]: Typo: `Contining` -> `Continuing`

Comment (mmcdaris) [reply jekyll/_docs/features/private-sourcemaps.md]: Thanks @thompiler, I've pushed a typo fix.

**Comment by mmcdaris** [comment]
Thanks for the reviews!

**Pull request merged**